spec/services/support_interface/candidate_journey_tracker_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'rails_helper'
 
 RSpec.describe SupportInterface::CandidateJourneyTracker, with_audited: true do
-  let!(:now) { Time.zone.now }
+  let!(:now) { Time.zone.now.change(usec: 0) }
```

Comment:
These tests were failing by nanoseconds. This works, but probably isn't the best solution. Normally, i'd just use the to be_within(1.second).of(Time.zone.now) matcher, but i was interested to see what people thought.

Reply:
I like `be_within` — it makes it clear that time is still moving. But in this case (and the other `usec:` specifications) I don't think either is necessary; the tests still pass without the `.change` call? Or maybe there's something different between our environments

Reply:
Yea they were passing locally it was failing on the builds.

Reply:
Getting the same issue locally. It looks like it's caused by a difference in precision between in-memory time and database time.

I like the idea outlined in the PR description re: a `current_time` helper. Interested to hear other opinions.

Reply:
I also like the `current_time` helper to mitigate this issue, which we can use across problematic specs. I'm curious how often this is happening, is it just this spec or across others (couldn't replicate locally)?

Reply:
there were a few that were failing on CI and a couple of others on the build. They were failing 100% of the time too.